src/nl802154.h

```diff
@@ -104,6 +106,12 @@ enum nl802154_attrs {
 
 	NL802154_ATTR_SUPPORTED_COMMANDS,
 
+	NL802154_ATTR_ADDRESS_MODE,
```

Comment:
cap_info couldnt really find defined anywhere for this application. addr_mode also not defined as a netlink attribute.